sdk/examples/client/block/custom_inputs.rs

```diff
@@ -19,39 +23,46 @@ async fn main() -> Result<()> {
     // This example uses secrets in environment variables for simplicity which should not be done in production.
     dotenvy::dotenv().ok();
 
-    let node_url = std::env::var("NODE_URL").unwrap();
-    let faucet_url = std::env::var("FAUCET_URL").unwrap();
+    let node_url = env::var("NODE_URL").unwrap();
+    let faucet_url = env::var("FAUCET_URL").unwrap();
+
+    let mut args = env::args().skip(1);
+    let amount = args.next().map(|s| s.parse::<u64>().unwrap()).unwrap_or(1_000_000u64);
 
-    // Create a client instance
```

Comment:
Why do you sometimes leave it and sometimes remove it? A comment can't hurt, even more in examples.

Reply:
Honestly I don't think this comment is necessary at all, because the code itself already says that it's creating a client. You may disagree, but I don't think comments should just repeat what the code already tells by itself.

Reply:
No, the code doesn't say it's creating a client, it says builder.finish.await which is not necessarily straightforward for everyone. Eve, more for people starting to use the SDK and maybe even potentially learning Rust. These are examples, we should add even more comments like that IMO.

Reply:
hmm, okay, I trust your judgement!